kong/db/strategies/off/init.lua

```diff
@@ -279,7 +274,7 @@ local function page_for_tags(self, size, offset, options)
   local rows, err
 
   rows, err, offset_token = page_for_prefix(self, prefix, size, offset_token,
-                                            options, true, dao.schema)
+                                            options, false, dao.schema)
```

Comment:
Why use `false` here? add some comments?

Reply:
the value of item key is item-lua-table-value. so we dont need to follow it.

Reply:
I think we dont need to add a comment, false is default value, and if the value is true, the code reader will find it means we need to follow, i think it is easy to understand. but if you review this code directly, not seeing page_for_prefix, it may be hard to understand what false means.